### DISCUSSION TITLE
I will use Japanese input elementary OS .

### DISCUSSION THREAD

**Discussion by akinobun** [elementary/feedback#114] [Q&A] [answered]
### Problem

I will use input Japanese in elementary OS . But I do not understand to use it .
So please teach me to input Japanese . Sorry . 

OS : elementary OS 7.1 Horus
Ubuntu 22.04.3 LTS Base

### Proposal

I do not know .

**Comment by ryonakano** [comment, marked as answer]
It's easy to input Japanese if you have common Japanese keyboard (106 keyboard) and installed elementary OS in Japanese.

Simply click the `jp` button in the top bar and  check the `日本語 (Mozc)` in the input method section:

![1](https://github.com/elementary/feedback/assets/26003928/877d2ca4-6286-4e42-8ef8-bbd0dbcca5f4)

It's ready to input Japanese. You can switch Hiragana input mode and direct input mode by Zenkaku/Hankaku key as you can do on Windows.
